web-console/src/druid-models/coordinator-dynamic-config.tsx

```diff
@@ -213,14 +213,16 @@ export const COORDINATOR_DYNAMIC_CONFIG_FIELDS: Field<CoordinatorDynamicConfig>[
     defaultValue: 100,
     info: (
       <>
-        The percentage of the total number of segments in the cluster that are considered every time
-        a segment needs to be selected for a move. Druid orders servers by available capacity
-        ascending (the least available capacity first) and then iterates over the servers. For each
-        server, Druid iterates over the segments on the server, considering them for moving. The
-        default config of 100% means that every segment on every server is a candidate to be moved.
-        This should make sense for most small to medium-sized clusters. However, an admin may find
-        it preferable to drop this value lower if they don&apos;t think that it is worthwhile to
-        consider every single segment in the cluster each time it is looking for a segment to move.
+        Deprecated. This will be phased out by the batched segment sampler. Only used if
```

Comment:
Can we document what the preferred alternative is here as well?

Reply:
could you clarify what you mean by this? I take it as you asking me to state that the preferred alternative is to set `useBatchedSegmentSampler == true`. so perhaps?

> Deprecated. Will eventually be phased out by the batched segment sampler. You can enable the batched segment sampler now by setting the dynamic Coordinator config, `useBatchedSegmentSampler`, to `true`. Note that if you choose enable the batched segment sampler, this config has no effect on balancing. If `useBatchedSegmentSampler == false`, this config defines the percentage of the total number of segments in the cluster that are considered every time a segment needs to be selected for a move. Druid orders servers by available capacity ascending (the least available capacity first) and then iterates over the servers. For each server, Druid iterates over the segments on the server, considering them for moving. The default config of 100% means that every segment on every server is a candidate to be moved. This should make sense for most small to medium-sized clusters. However, an admin may find it preferable to drop this value lower if they don't think that it is worthwhile to consider every single segment in the cluster each time it is looking for a segment to move.|100|

Reply:
Sorry for ambiguity. Yes, encouraging using `useBatchedSegmentSampler` instead will be enough. The one in your comment looks good to me.